apps/src/pythonlab/PythonConsole.tsx

```diff
@@ -19,23 +19,27 @@ const PythonConsole: React.FunctionComponent = () => {
   const {loading, data} = useFetch('/api/v1/users/current/permissions');
   const dispatch = useDispatch();
 
-  const handleRun = () => {
+  const handleRun = (runTests: boolean) => {
     const parsedData = data ? (data as PermissionResponse) : {permissions: []};
     // For now, restrict running python code to levelbuilders.
     if (parsedData.permissions.includes('levelbuilder')) {
-      dispatch(appendSystemMessage('Running program...'));
-      let mainFound = false;
       if (source) {
-        const code = getFileByName(source.files, MAIN_PYTHON_FILE)?.contents;
-        if (code) {
-          mainFound = true;
-          runPythonCode(code, source);
+        if (runTests) {
```

Comment:
Long-term, I think this would be slightly easier to read if we kept these conditions "flatter", but no need to change it for this PR!

Reply:
Since I was already cleaning things up I flattened this up a bit, lmk if that's what you had in mind!

Reply:
Looks great, thanks!